exporter/exporterhelper/internal/bounded_memory_queue_test.go

```diff
@@ -127,9 +129,16 @@ func TestShutdownWhileNotEmpty(t *testing.T) {
 	q.Produce(newStringRequest("i"))
 	q.Produce(newStringRequest("j"))
 
+	go func() {
```

Comment:
Please comment why async, if it takes more than few seconds to understand why it needs a comment :) (and yes, it took me some time to understand why executing async).

Reply:
sure thing